lib/mtl/rails/card_file_presenter.rb

```diff
@@ -17,6 +17,7 @@ def render(filename, href, params = {})
                      title: params[:title] || filename,
                      target: '_blank',
                      class: ['card-panel', params[:preview] ? 'card-panel-image' : nil],
+                     data: { mtl_document_modal: 'open', mtl_document_name: filename },
```

Comment:
shall we make this optional ? Aka you're able to say if you want it to open a modal or not? For example, with cardfiles used for pre-uploaders and such